src/lib/components/EventStore/index.js

```diff
@@ -80,12 +83,19 @@ class EventStore extends HTMLElement {
         nextPendingDay = day;
       }
 
-      // Is this day actually active (within exact time range)?
-      const isActive = now >= bufferStart && now <= bufferEnd;
+      // Is this day active (within the buffer time range)?
+      const isActive = now >= bufferStart && now < bufferEnd;
       if (isActive && this._activeDay !== day) {
         this._activeDay = day;
         change = true;
       }
+
+      // Is this the active day for chat (within the actual time range)?
+      const isActiveChat = now >= actualStart && now < actualEnd;
```

Comment:
my hunch is we'll want to buffer this by some amount (10 minutes?) as well to let folks wrap up their conversations. But we can go with this for now and ask Harleen what she wants.

Reply:
I've made this ten minutes, and updated all the duration stuff to be in minutes to correspond with that.